src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();
+    private static final String COLOR = new ColorSupplier().getRandomColor();

```

Comment:
The COLOR variable should not be static because it will be initialized only once and all figures will have the same color. Instead, you should generate a new color for each figure inside the `getRandomFigure()` method.